bank.rb

```diff
@@ -0,0 +1,39 @@
+require_relative 'account'
+# SauceBank
+class Bank
+  attr_accessor :accounts
+  def initialize
+    @accounts ||= load_accounts
+  end
+
+  def load_accounts
+    []
+  end
+
+  def find_account(name)
+    @accounts.select do |acct|
+      acct.owner.name.downcase == name.downcase
+    end
+  end
+
+  def has_account?(name)
+    query = find_account(name)
+    !query.empty?
+  end
+
+  def create_account(name)
+    new_acct = Account.new({ owner: name })
```

Comment:
The bank shouldn't be responsible for creating the account.  It should have less knowledge of other classes, and very little knowledge of how those classes operate.